myocamlbuild.ml

```diff
@@ -7,6 +7,9 @@ let system_support_lib = match os with
 | "Linux\n" -> [A "-cclib"; A "-lrt"]
 | _ -> []
 
+let lib_a = Ocamlbuild_pack.Ocamlbuild_config.a
+let clock_stubs_lib = "src-clock/os/libptime_clock_stubs." ^ lib_a
```

Comment:
Not saying this is wrong. But could you follow [what is done](https://github.com/dbuenzli/mtime/blob/184ae332fee883631735cf83a55866d40c5c0fb1/myocamlbuild.ml#L25-L28) in `mtime`. That way everything fails according to same reasons when it does :–)

Reply:
No problem. Fixed